scripts/issue/690/claim_and_deposit_tcl.py

```diff
@@ -0,0 +1,108 @@
+from pycoingecko import CoinGeckoAPI
+from brownie import accounts
+from great_ape_safe import GreatApeSafe
+from helpers.addresses import r
+
+# THRESHOLDS FOR TARGET OPS
+MAX_GRAVI_MANTISSA = 25_000e18
+MAX_WBTC_MANTISSA = int(6.5e8)
+
+WBTC_WHALE = "0x218B95BE3ed99141b0144Dba6cE88807c4AD7C09"
+DIGG_WHALE = "0x95EEC544A7Cf2e6a65A71039d58823f4564A6319"
+
+def main(simulation="false"):
+    vault = GreatApeSafe(r.badger_wallets.treasury_vault_multisig)
+    voter = GreatApeSafe(r.badger_wallets.treasury_voter_multisig)
+    vault.init_aura()
+    vault.init_balancer()
+
+    # tokens involved
+    wbtc = vault.contract(r.treasury_tokens.WBTC)
+    digg = vault.contract(r.treasury_tokens.DIGG)
+    aura = vault.contract(r.treasury_tokens.AURA)
+    bal = vault.contract(r.treasury_tokens.BAL)
+    graviaura = vault.contract(r.sett_vaults.graviAURA)
+    # https://app.balancer.fi/#/pool/0x8eb6c82c3081bbbd45dcac5afa631aac53478b7c000100000000000000000270
+    bpt_40_wbtc_40_digg_20_graviaura = vault.contract(
+        r.balancer.bpt_40wbtc_40digg_20graviaura
+    )
+
+    if simulation == "true":
+        # Fund Vault with wBTC and Digg until thresher is executed
+        wbtc_whale = accounts.at(WBTC_WHALE, force=True)
+        digg_whale = accounts.at(DIGG_WHALE, force=True)
+
+        wbtc.transfer(vault.address, MAX_WBTC_MANTISSA, {"from": wbtc_whale})
+        digg.transfer(vault.address, 14e9, {"from": digg_whale})
+        
+    vault.take_snapshot([wbtc, digg, aura, bal, graviaura])
+    voter.take_snapshot([bal, aura])
+
+    # grab prices from coingecko
+    ids = ["aura-finance", "wrapped-bitcoin", "digg"]
+    prices = CoinGeckoAPI().get_price(ids, "usd")
+
+    # claim rewards
+    vault.aura.claim_all_from_booster()
+
+    graviaura_bal = graviaura.balanceOf(vault)
+    aura_bal = aura.balanceOf(vault)
+
+    ppfs = graviaura.getPricePerFullShare() / 1e18
+    graviaura_needed = MAX_GRAVI_MANTISSA - graviaura_bal
+    aura_to_deposit = graviaura_needed * ppfs
+    remaining_aura = aura_bal - aura_to_deposit
+
+    # deposit aura into into graviaura to have at least 25k gravi available
+    aura.approve(graviaura, aura_to_deposit)
+    graviaura.deposit(aura_to_deposit)
+    graviaura_mantissa = graviaura.balanceOf(vault)
+
+    # send BAL and remaining of AURA
+    bal.transfer(voter, bal.balanceOf(vault))
+    aura.transfer(voter, remaining_aura)
```

Comment:
wouldnt it be better to `graviaura.depositFor(voter, remaining_aura)` here?

Reply:
idea is to lock as vanilla in voter, not to deposit in graviaura